site/content/blog/2022-01-01-whats-new-in-svelte-january-2022.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "What's new in Svelte: January 2022"
+description: "Faster builds with SvelteKit and a much anticipated REPL feature"
+author: Daniel Sandoval
+authorURL: https://desandoval.net
+---
+
+Happy new year, Svelte Community! Lots to share this month across Svelte, SvelteKit, Language Tools and the Showcase. Thanks to everyone who made 2021 a great year to use Svelte. Looking forward to the next one 🚀
+
+## What's new in `svelte/*`
+- [Svelte 3.44.3](https://github.com/sveltejs/svelte/blob/master/CHANGELOG.md#3443) is out with a few bug fixes in the binding and loop code
+- Svelte Language Tools has introduced support for the then/catch shorthands from Svelte 3.41 and TypeScript's "go to" functionality ([105.8.0 and later](https://github.com/sveltejs/language-tools/releases/tag/extensions-105.8.0))
+- After the [update to Vite 2.7](https://github.com/sveltejs/kit/pull/3018), SvelteKit users are [reporting significant performance improvements](https://www.reddit.com/r/sveltejs/comments/rljhfc/sveltekit_massive_compiler_improvement_by/)
+- `@sveltejs/adapter-static` for SvelteKit now has a `precompress` option to make brotli compression of assets and pages easier to do out of the box ([#3079](https://github.com/sveltejs/kit/pull/3079))
+- Concurrency mode in SvelteKit will now render pages in parallel ([#3120](https://github.com/sveltejs/kit/pull/3120)). It is enabled by default in `1.0.0-next.205` and later
+- The Svelte REPL got a nice upgrade as well - letting you delete saved REPLs. Try it out by logging in at [svelte.dev/apps](https://svelte.dev/apps)
+
```

Comment:
other major sveltekit improvements include:
- loading CSS before JS for improved page performance (https://github.com/sveltejs/kit/commit/d138efe21692f5925f1e89afc0a33f42d6a1a711)
- ability to disable service worker registration to do your own custom registration (https://github.com/sveltejs/kit/pull/2988)
- SSR route-splitting (https://github.com/sveltejs/kit/pull/2931)
- SvelteKit server will now automatically restart when the config files is changed (https://github.com/sveltejs/vite-plugin-svelte/pull/237)

Reply:
Awesome! Added. Thanks!